### ISSUE TITLE
Black shirt for ink splatter node 

### ISSUE THREAD

**Issue by superlink64** [PastaOwO/Squidnodes#3]
so i followed the steps for how to use the ink splatter bsdf, but the outcome was this
<img width="122" alt="image" src="https://user-images.githubusercontent.com/100834090/213596921-9ba68071-6830-4db6-9c2c-fb00fd643f0e.png">

**Comment by PastaOwO** [comment]
There are 3 main issues i can think of off the top of my head:

   1. Alpha clipping is disabled on the material,
   2. The inksplatter mesh is clipping into the shirt mesh, or:
   3. You might be using the same mesh for both the shirt and the ink.

Let me know if you need any other information!

Kind regards,
Pasta

**Comment by superlink64** [comment]
I'm pretty sure alpha clipping is on, I did duplicate my shirt mesh, and if I don't have the duplicate shirt on top of the original one, then the duplicate appears invisible. Here, I'll send my file to you. 
[splatoon 3.zip](https://github.com/PastaOwO/Squidnodes/files/10476080/splatoon.3.zip)

**Comment by PastaOwO** [comment]
Can you resend that but make sure that "pack external files" is enabled?
The textures are broken.

Kind regards,
Pasta

**Comment by superlink64** [comment]
[splatoon 3.zip](https://github.com/PastaOwO/Squidnodes/files/10476559/splatoon.3.zip)
ok, now? (I'm a blender noob)

**Comment by PastaOwO** [comment]
Yup, I got it. There are actually a few issues here, lots of the textures
have the wrong color space.
I'm gonna fix those issues for you with squidnodes nodes and other things,
and you can compare the two versions to see what changed.

Kind regards,
Pasta

**Comment by superlink64** [comment]
done yet?
